autopush/webpush_server.py

```diff
@@ -359,8 +361,9 @@ def lookup_user(self, hello):
         # type: (Hello) -> (Optional[JSONDict], JSONDict)
         flags = dict(
             message_month=None,
-            rotate_message_table=False,
+            check_storage=False,
```

Comment:
nit: can we not change the order of these? It's not like they're in alphabetical order anyway, and it's generally better to put new items at the end of lists to avoid accidental flag set.

Reply:
When creating a dict? I understand it in keyword args to API methods to add it to the end, but this just makes a dict.

Reply:
fair, but since we do specify a set of args that match these, I thought it might be nice for consistency.